recipe/meta.yaml

```diff
@@ -15,9 +15,9 @@ source:
 
 build:
   skip: True   # [linux and py27]
-  number: 0
+  number: 1
   script:
-    - {{ PYTHON }} -m pip install --upgrade pip  # [win]
+    - {{ PYTHON }} -m pip install --upgrade pip
```

Comment:
why do you need to do this?

Reply:
you mean update pip at all? I don't really remember, I'll get rid of it.

Reply:
using pip to install anything other than the package you're building  will include things you don't want in your package

Reply:
thanks, it's removed now 👍